### ISSUE TITLE
On "Toggle caption on" focus should move to the caption

### ISSUE THREAD

**Issue by lauriii** [ckeditor/ckeditor5#12364]
## 📝 Provide detailed reproduction steps (if any)

1. Enable images + image captions
2. Press "Toggle caption on" for an image

### ✔️ Expected result

Focus should move to the caption.

### ❌ Actual result

Focus is not moved to the caption but to the button.

## 📃 Other details

Tested on HEAD, https://github.com/ckeditor/ckeditor5/commit/f811d4961665b2c9954a35aad5a200b70e894485

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by mlewand** [comment]
That's weird, it has already been reported as ckeditor/ckeditor5#12165 and has been closed in the current iteration.

However you mentioned that you have checked it on ckeditor/ckeditor5@f811d49 which means the fix should be already there.

@ckeditor/qa-team can you double check if it works as expected?

**Comment by LukaszGudel** [comment]
Hi @lauriii,

I am not able to reproduce your issue on `f811d49` commit. It works as expected for me. Could you check the demo on this site: https://ckeditor5.github.io/docs/nightly/ckeditor5/latest/features/images/images-overview.html? Editors there are rebuilt every day using the current `master` branch.

If you can reproduce the issue on our nightly docs as well, could you please share more details like os and browser version?

**Comment by lauriii** [comment]
Sorry about the noise. I have confirmed both on the nightly docs and by using 35.1.0 that this has been addressed. I believe I may have had an older version of CKEditor 5 image plugin on the build.
